astro/src/layouts/EscapeRoomLayout.astro

```diff
@@ -0,0 +1,470 @@
+    <slot name="head" />
+  </head>
+
+  <body>
+    <!-- Custom Header -->
+    <header class="escape-room-header">
+      <nav class="navbar navbar-expand-lg navbar-dark bg-transparent">
+        <div class="container">
+          <a class="navbar-brand" href="/services/escape-room/">
+            <div class="logo-container">
+              <Image
+                src={escapeRoomLogo}
+                alt=""
+                width="40"
+                height="40"
+                class="logo-image"
+                aria-hidden="true"
+              />
+            </div>
+            <div class="brand-text">
+              <span class="accessible">The Accessible</span>
+              <span class="escape-room">Escape Room</span>
+            </div>
+          </a>
+          <button
+            class="navbar-toggler"
+            type="button"
+            id="navToggler"
+            aria-controls="navbarNav"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav ms-auto">
+              <li class="nav-item">
+                <a class="nav-link" href="/">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="/services">Services</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="/about">About</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="/volunteer">Join us</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <main id="main-content">
+      <div class="content-spacer"></div>
+      <slot name="header" />
+      <slot />
+    </main>
+
+    <!-- Custom Footer -->
+    <footer class="escape-room-footer">
+      <div class="container">
+        <div class="row g-4">
+          <div class="col-md-4">
+            <h3 class="h5">About Accessible Community</h3>
+            <p>
+              We help organizations make their digital content accessible to people with disabilities.
+            </p>
+          </div>
+          <div class="col-md-4">
+            <h3 class="h5">Quick Links</h3>
+            <ul class="list-unstyled">
+              <li><a href="/services">Services</a></li>
+              <li><a href="/about">About Us</a></li>
+              <li><a href="/volunteer">Join us</a></li>
+            </ul>
+          </div>
+          <div class="col-md-4">
+            <h3 class="h5">Contact Us</h3>
+            <p>
+              Have questions about our services?<br />
+              <a href="mailto:info@accessiblecommunity.org">info@accessiblecommunity.org</a>
+            </p>
+          </div>
+        </div>
+        <hr />
+        <div class="text-center">
```

Comment:
instead of another div here can we just place the text-center on paragraph since it's a block level elements

Reply:
@ashimupd Thoughts on placing text-center just on the p itself?